src/i18n/General.js

```diff
@@ -260,7 +260,7 @@ export default {
     en: 'Disabled',
   },
   SSOEnable: {
-    zh: '启用 {backend} SSO',
+    zh: '启用{backend} SSO',
```

Comment:
Because a space was removed here; the reason for removing the space is that when the `backend` is in Chinese, it's not appropriate to have a space here.